spec/support/with_default_schedules.rb

```diff
@@ -15,7 +15,7 @@
       npq_specialist_schedule: %w[npq-specialist-spring npq-specialist-autumn],
       npq_leadership_schedule: %w[npq-leadership-spring npq-leadership-autumn],
       npq_aso_schedule:        %w[npq-aso-december],
```

Comment:
What about these 3 above?

Reply:
good question, `npq_aso_schedule` we don't pick the schedule according to time, so should be ok, the other two include all the schedules we pick according to the time so hopefully also will be ok

Reply:
Ah ok! I think those are all the variants. Forget about my comment.

Reply:
> Wouldn't it be better to make it Date.current dependent?
To unblock main, if this passes I'm hoping we can merge, and I'll spend more time on your suggestion to avoid having to set up all schedules and rely on the defaults at the time, as that will require more time and I don't want to block the pipeline. Does that sound ok?